app/index.js

```diff
@@ -0,0 +1,8 @@
+import express from "express";
+const port = process.env.PORT || 3000;
+const app = express();
+
+app.listen(port, (error) => {
```

Comment:
Totally wrong as for me.
`server.listen(PORT, () => {console.log("Listening...!");}).on("error", (err) => {throw Error("MUUUUUUUUUU", err);});`

Reply:
Besides, why not set headers here?

Reply:
Change handling and also add headers